src/Microsoft.AspNetCore.Server.Testing/Deployers/ApplicationDeployer.cs

```diff
@@ -19,23 +19,23 @@ public abstract class ApplicationDeployer : IApplicationDeployer
     {
         private readonly Stopwatch _stopwatch = new Stopwatch();
 
+        protected readonly string DotnetCommandName = "dotnet";
```

Comment:
not sure if I understand why it makes a difference. protected on a public class is like public - anyone can expose it, and these values are basically consts.